.github/workflows/positron-python-ci.yml

```diff
@@ -131,7 +131,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         # Run the tests on the oldest and most recent versions of Python.
-        python: ['3.9', '3.13']
+        python: ['3.9', '3.13', '3.14']
```

Comment:
adding this for now instead of replacing 3.13, until https://github.com/posit-dev/positron/issues/9654